src/config.nim

```diff
@@ -40,7 +41,13 @@ proc getConfig*(path: string): (Config, parseCfg.Config) =
     enableRss: cfg.get("Config", "enableRSS", true),
     enableDebug: cfg.get("Config", "enableDebug", false),
     proxy: cfg.get("Config", "proxy", ""),
-    proxyAuth: cfg.get("Config", "proxyAuth", "")
+    proxyAuth: cfg.get("Config", "proxyAuth", ""),
+    cookieHeader: cfg.get("Config", "cookieHeader", ""),
```

Comment:
@PrivacyDevel This is inconvenient to use when the values can change around if using it in a docker container.
Please change it to
```nim
cfg.get("Config", "cookieHeader",  getEnv("cookieHeader", "") )
cfg.get("Config", "xCsrfToken",  getEnv("xCsrfToken", "") )
```

Reply:
When running docker containers through nice web frontends this makes it easy to change the files without needing to screw around with the twitter conf and getting the `cookieHeader` all properly escaped for the `.conf`, because you can just paste it into a text box and it handles the escaping.